Directory.Build.props

```diff
@@ -67,51 +76,9 @@
         <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
       </PropertyGroup>
       <ItemGroup>
-        <PackageReference Include="Microsoft.SourceLink.AzureRepos.Git" Version="1.0.0" PrivateAssets="All" />
```

Comment:
Why did you remove this?

Reply:
We aren't using Azure Repos. This package was redundant if that's the case.

Reply:
@azchohfi do we have any mirrors still hosted in Azure elsewhere? I thought we still may need this?

Reply:
We build it on ADO, but the source is always on Github. I believe we are good removing this.